tests/mocks/MockHttpRequester.hh

```diff
@@ -26,6 +26,11 @@ public:
         return ra::etoi(response.StatusCode());
     }
 
+    bool IsRetryable(_UNUSED unsigned int nStatusCode) const noexcept
```

Comment:
I found out that not putting a parameter name doesn't trigger the unused formal parameter warning. I'm guessing that's why they called it `maybe_unused`. Parameter name could be commented out the have the same effect.

Reply:
The main reason I did that was for configurations. Like if for some reason some using that file didn't have C++17, it'll do something else (or nothing if no alternative exists).

Might be better to define as `_MAYBE_UNUSED`?